README.md

```diff
@@ -22,6 +24,8 @@ bundle exec rspec
 bin/rails server
 ```
 
+> **Note**: The documentation uses Ruby 2.7.2. You also need Node installed. The current LTS (long term support) version should be ok.
```

Comment:
How about pointing to https://github.com/buildkite/docs/blob/main/.ruby-version for the ruby version instead? :p

Reply:
Yeah I was on the fence between friendly and maintainable. I shall edit.

Reply:
Write a custom ssg to insert the value at commit time?